.ci-build-without-test.sh

```diff
@@ -42,7 +42,7 @@ export PATH=$AFU/scripts:$JAVA_HOME/bin:$PATH
 /tmp/plume-scripts/git-clone-related opprop checker-framework ${CHECKERFRAMEWORK}
 
 # This also builds annotation-tools
-(cd $CHECKERFRAMEWORK && checker/bin-devel/build.sh downloadjdk)
+(cd $CHECKERFRAMEWORK && ./gradlew assemble)
```

Comment:
Is `assembleForJavac` enough? That target is faster, as it doesn't build javadoc.

Reply:
Yes, I think this will works. I have updated the PR.